### ISSUE TITLE
Error with MdiIcons.fromString

### ISSUE THREAD

**Issue by ItsJimi** [ziofat/material_design_icons_flutter#14]
Hi,
I'm trying to dynamically add an icon but it can be wrong and when i use `MdiIcons.fromString()` with a wrong icon, i got this error:
```
The method 'toRadixString' was called on null.
Receiver: null
Tried calling: toRadixString(16)
```
I think it could be nice to return something like null if the icon doesn't exist.
I made a simple pull request to handle error, i don't know if it's the right way.
https://github.com/ziofat/material_design_icons_flutter/pull/15

Thanks

**Comment by ziofat** [comment]
The PR looks good to me. I will publish it soon.

**Comment by ziofat** [comment]
Published in `3.3.4595`, thanks for your contribution.
